mkdocs.yml

```diff
@@ -4,6 +4,8 @@ pages:
   - [styleguide.md, Style Guide]
   - [gettingstarted/index.md, Getting Started, Home]
   - [gettingstarted/structuring.md, Getting Started, Structuring Your Mod]
+  - [blockstates/introduction.md, Models & Rendering, Blockstates]
+  - [blockstates/forgeBlockstates.md, Models & Rendering, Blockstates, Forges Blockstates]
```

Comment:
Ok, after further investigating it is using a number as the link label which causes the error....bizzare. Using a name like `forge` i.e. 

```
... [Forges Blockstate Json][forge] allows you to do so in 1.8.
[forge]: forgeBlockstates.md "Forges Blockstate Json"
```

seems to work fine.